kokoro/ubuntu/release.sh

```diff
@@ -59,27 +59,21 @@ if [ "$SOURCE_VERSION" != "$TAG_VERSION" ]
         exit 1 # terminate and indicate error
 fi
 
-echo "Installing aktau/github-release.."
+echo "Installing github-release"
 go get github.com/itchio/gothub
 cd github/google-cloud-intellij
 echo "Building plugins"
 ./gradlew buildPlugin
 
 echo "Creating Github release for tag: $GIT_TAG_NAME"
-
```

Comment:
I think maybe the version in master is incorrect:
the echo statement says its pull aktau/github-releases, but then the command pulls from itchio/gothub. 

Also, should I have kepts all those exports and --name lines below?